Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -1819,6 +1819,11 @@ void TestController::didReceiveSynchronousPageMessageFromInjectedBundleWithListe
     static_cast<TestController*>(const_cast<void*>(clientInfo))->didReceiveSynchronousMessageFromInjectedBundle(messageName, messageBody, listener);
 }
 
+void TestController::didReceiveAsyncMessageFromInjectedBundle(WKPageRef page, WKStringRef messageName, WKTypeRef messageBody, WKMessageListenerRef listener, const void* clientInfo)
```

Comment:
nit - including "PageMessageFromInjectedWithListener" in the function name would match the function name of the sync version above. also `page` is an unused parameter.